buildSrc/src/main/java/org/opensearch/gradle/pluginzip/Publish.java

```diff
@@ -72,10 +82,8 @@ public void apply(Project project) {
             }
             Task publishPluginZipPublicationToZipStagingRepository = project.getTasks()
                 .findByName("publishPluginZipPublicationToZipStagingRepository");
-            if (validatePluginZipPom != null) {
```

Comment:
we might still need this @reta 
The reason I have added this because I got the following warning
Error:
```
> Task :validatePluginZipPom
Execution optimizations have been disabled for task ':validatePluginZipPom' to ensure correctness due to the following reasons:
  - Gradle detected a problem with the following location: '/usr/share/opensearch/git/opensearch-plugin-template-java/build/distributions/rename-unspecified.pom'. Reason: Task ':validatePluginZipPom' uses this output of task ':generatePomFileForNebulaPublication' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.4.2/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```

Reply:
The `validatePluginZipPom` is above, I did not touch it: https://github.com/opensearch-project/OpenSearch/pull/3252/files/7720ae0d0412a97b77d1077be08cca6a95eedecf#diff-2a5851da2408bbeb8733433421917f253614fffe49449da0dd315a15ed8cdf74R80

This condition should be for `publishPluginZipPublicationToZipStagingRepository`, not `validatePluginZipPom`

Reply:
lol my bad, this formatting looks like it was removed, yes I can see those 2 conditions are not touched.